pkgs/by-name/kp/kplex/package.nix

```
@@ -0,0 +1,27 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+}:
+stdenv.mkDerivation rec {
+  pname = "kplex";
+  version = "1.4";
+  src = fetchFromGitHub {
+    owner = "stripydog";
+    repo =  "kplex";
+    rev = "v${version}";
+    hash = "sha256-sps9l238hGLJ673kewFH8fOJw0HphEkZbJ+VUIzxC+o=";
+  };
+
+  installPhase = ''
+    install -D -m 0555 kplex $out/bin/kplex
+  '';
```

The install changes are optional, but the hooks should be there.

```suggestion
  installPhase = ''
    runHook preInstall
    install -Dm555 kplex -t $out/bin
    runHook postInstall
  '';
```